evalml/pipelines/time_series_regression_pipeline.py

```diff
@@ -10,9 +7,7 @@
 )
 
 
-class TimeSeriesRegressionPipeline(
-    RegressionPipeline, metaclass=TimeSeriesPipelineBaseMeta
-):
+class TimeSeriesRegressionPipeline(TimeSeriesPipelineBase):
```

Comment:
Can you please mark that `TimeSeriesRegressionPipeline` does not inherit from `RegressionPipeline` as a breaking change?

Reply:
sure thing 😁